hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -110,6 +114,7 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 			in.readLong();
 			in.readLong();
 		}
+		nftsOwned = in.readLong();
```

Comment:
This would break merkle release compatibility. You have to add new RELEASE_VERSION and `encode/decode` the nftsOwned only after the version is the new release. See MerkleToken for reference

Reply:
Does that mean I have to do the same not only for encode/decode, but also for the copy() function where essentially we `copy` the object and only copy nftsOwned if the version is right?